content/utilities/tiny-leaderboard/plugin.js

```diff
@@ -0,0 +1,171 @@
+// Tiny Leaderboard
+//
+// Shows a tiny leaderboard with a timer (until the round ends)
+// Shows the 5 people above and below you on the board
+// Updates every 60 seconds
+
+const updateTimeInSeconds = 60;
+const leaderboardRange = 5; // 5 people above and below you
+
+// ---------------------------------------------------------------
+
+// https://github.com/darkforest-eth/client/blob/19f2547dce33f3274afa86945e96fbfebc7693f5/src/Frontend/Views/Leaderboard.tsx#L119
+const roundEndTimestamp = '2021-07-07T21:00:00-07:00';
+const roundEndTime = new Date(roundEndTimestamp).getTime();
+
+function formatDuration(durationMs) {
+	if (durationMs < 0) return '';
+	const hours = Math.floor(durationMs / 1000 / 60 / 60);
+	const minutes = Math.floor((durationMs - hours * 60 * 60 * 1000) / 1000 / 60);
+	const seconds = Math.floor((durationMs - hours * 60 * 60 * 1000 - minutes * 60 * 1000) / 1000);
+	return (
+		timestampSection(hours) + ':' + timestampSection(minutes) + ':' + timestampSection(seconds)
+	);
+}
+
+function timestampSection(value) {
+	return value.toString().padStart(2, '0');
+}
+
+function getStrTimeUntilRoundEnds() {
+	const timeUntilEnd = roundEndTime - new Date().getTime();
+	if (timeUntilEnd <= 0) return "00:00";
+	else return formatDuration(timeUntilEnd);
+}
+
+// ---------------------------------------------------------------
+
+function formatNumber(num, decimalCount = 1) {
+	num = parseInt(num);
+	if (num < 1000) return num;
+	if (num < 1000000) return (num / 1000).toFixed(decimalCount) + "k";
+	if (num < 1000000000) return (num / 1000000).toFixed(decimalCount) + "m";
+	if (num < 1000000000000) return (num / 1000000000).toFixed(decimalCount) + "b";
+	return (num / 1000000000000, 2).toFixed(decimalCount) + "t";
+}
+
+async function downloadLeaderboard() {
+	return fetch('https://api.zkga.me/leaderboard')
+		.then(response => response.json())
+}
+
+// return example: 'hsl(285,100%,70%)'
```

Comment:
```suggestion
```

Reply:
i put it in there to know in what kind of format the color is returned by the function